src/win32/thread.c

```diff
@@ -240,14 +215,26 @@ int pthread_rwlock_wrunlock(pthread_rwlock_t *lock)
 	return 0;
 }
 
-int pthread_rwlock_destroy(pthread_rwlock_t *lock)
+int git_rwlock_free(git_rwlock *lock)
 {
 	if (!win32_srwlock_initialize)
 		DeleteCriticalSection(&lock->native.csec);
 	git__memzero(lock, sizeof(*lock));
 	return 0;
 }
 
+int git_num_processors(void)
```

Comment:
Yes, it is currently unused, as are some other functions (e.g. `git_cond_broadcast`, which is even unimplemented for Windows as of now). I'm all in for removing unused cruft